### PR TITLE
DRUP-1043: #ready adds mobile css and menu title

### PR DESCRIPTION
**Pull request by emb03**
@z3cka syncing master with stage (features re-rolled from prod), plz merge, thx!

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.info [hunk 1]
```diff
@@ -111,4 +111,8 @@ features[views_view][] = bookplates_landing_page
 features[views_view][] = bookplates_single_node
 features_exclude[dependencies][page_manager] = page_manager
 features_exclude[dependencies][fe_block] = fe_block
+<<<<<<< HEAD
 mtime = 1471449968
+=======
+mtime = 1475172467
+>>>>>>> cb2a229... DRUP-1043: adds mobile css and menu title
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.info]: pick one of the above `mtime` values. Just commit to the same branch and push; that will update this PR and I will squash when merging.

**Comment by emb03** [comment]
@z3cka dang it

**Comment by z3cka** [comment]
@emb03 no biggie :-D

**Comment by z3cka** [comment]
I would fix it myself, but I didn't see a way to add my own changes on top without going through the whole workflow.

**Comment by emb03** [comment]
@z3cka okay, fixed.

**Comment by z3cka** [comment]
deploying now

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**